css/style.css

```diff
@@ -1 +1 @@
;width:100%;bottom:30px;text-transform:uppercase;text-align:center;padding-bottom:5px;color:#2f3640;font-family:Montserrat;font-size:14px;font-weight:500;transition:ease-in-out 0.3s}.btn--add-to-cart:after{content:"";position:absolute;width:30px;top:7px;right:50px;height:2px;opacity:0.7;border-bottom:1px solid #2f3640}.btn--add-to-cart:before{content:"";position:absolute;width:30px;top:7px;left:50px;height:2px;opacity:0.7;border-bottom:1px solid #2f3640}.card-price{position:absolute;top:20px;right:20px;font-family:"Roboto", sans-serif;font-size:30px;font-weight:bold;color:#4f4f4f}.card-container:hover{box-shadow:0px 0px 35px rgba(0,0,0,0.45)}.card-container:hover .btn--add-to-cart{opacity:0.7}.card-image-container{position:absolute;top:0;width:100%;height:90%;margin:0 auto}.card-image-container>img{display:block;margin:0 auto;max-width:100%;height:100%;object-fit:contain;transform:scale(0.5);transition:ease-in-out 0.6s;opacity:0.9}.cart-btn{opacity:0;position:absolute;top:16px;padding:10px;left:20px;font-size:22px;color:#bdbdbd;transition:ease-in-out 0.3s}.card-container:hover .cart-btn{opacity:1}.container{width:100%;display:grid;grid-template-rows:repeat(1, 1fr);grid-template-columns:repeat(1, 1fr);justify-content:space-around;gap:2rem}@media only screen and (min-width: 37.5em){.container{grid-template-columns:repeat(2, 1fr)}}@media only screen and (min-width: 56.25em){.container{grid-template-columns:repeat(3, 1fr)}}@media only screen and (min-width: 75em){.container{grid-template-columns:repeat(4, 1fr)}}.card-container{margin-top:5rem;justify-self:center}@media only screen and (min-width: 37.5em){.card-container{margin-top:0}}*{box-sizing:border-box}body{margin:0px;font-family:"segoe ui"}.nav{z-index:100;height:50px;width:100%;background-color:#060b26;position:relative;justify-content:space-between}.nav-header{display:inline;line-height:3rem}.nav-title{display:inline-block;font-size:22px;color:#fff;padding:10px}.nav-btn{display:none}@media (max-width: 600px){.nav-btn{display:inline-block;position:absolute;right:0px;top:0px}}@media (max-width: 600px){.nav-btn label{display:inline-block;width:50px;height:50px;padding:13px}}@media (max-width: 600px){.nav-btn label span{display:block;width:25px;height:10px;border-top:2px solid #eee}}.nav-links{display:inline;float:right;font-size:1.6rem}.nav-links a{display:inline-block;padding:13px 10px 13px 10px;text-decoration:none;color:#efefef}@media (max-width: 600px){.nav-links a{display:block;width:100%}}.nav-links a:hover{background-color:rgba(0,0,0,0.3)}@media (max-width: 600px){.nav-links{position:absolute;display:block;width:100%;background-color:#060b26;height:0px;transition:all 0.3s ease-in;overflow-y:hidden;top:50px;left:0px}}.nav #nav-check{display:none}@media (max-width: 600px){.nav>#nav-check:not(:checked) ~ .nav-links{height:0px}.nav>#nav-check:checked ~ .nav-links{height:calc(100vh - 50px);overflow-y:auto}}footer p{padding:4rem 0;text-align:center;font-size:2rem;color:#060b26;margin-top:20rem}
```

Comment:
It seems like a minified generated css, usually we try not to commit built files.